EtoolsDataTableRow.js

```diff
@@ -50,7 +50,7 @@ export class EtoolsDataTableRow extends LitElement {
           display: flex;
           flex-direction: row;
           align-items: center;
-          padding: var(--list-row-wrapper-padding, 0 24px 0 0);
+          padding-inline: var(--list-row-wrapper-padding-inline, 0 24px);
```

Comment:
Rename is probably unnecessary as it triggers the need to update in parent apps also